agent/src/test/scala/za/co/absa/atum/agent/AtumContextTest.scala

```diff
@@ -108,7 +108,7 @@ class AtumContextTest extends AnyFlatSpec with Matchers {
     val measurements = Seq(
       MeasurementProvided(RecordCount("col"), 1L),
       MeasurementProvided(SumOfValuesOfColumn("col"), BigDecimal(1)),
-      MeasurementProvided.forCustomMeasure("customMeasureName", "col", BigDecimal(1))
+      MeasurementProvided("customMeasureName", "col", BigDecimal(1))
```

Comment:
I'm thinking that maybe the constructor of `CustomMeasure` don't have to be private, I think this seems nicer:

MeasurementProvided(CustomMeasure("customMeasureName", "col"), BigDecimal(1))

Reply:
Ok. Changed.